Tasks/AzureRmWebAppDeployment/operations/KuduServiceUtility.ts

```diff
@@ -127,6 +127,47 @@ export class KuduServiceUtility {
         }
     }
 
+    public async zipDeploy(packagePath: string, appOffline: boolean, customParameters?: Array<string>): Promise<void> {
```

Comment:
appOffline should not be a mandatory input.

Reply:
done. thanks